### DISCUSSION TITLE
Footer Setting

### DISCUSSION THREAD

**Discussion by nicogithub22** [globaleaks/globaleaks-whistleblowing-software#3389] [Q&A]
### What version of GlobaLeaks are you using?

4.10.18

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

I have added a text under Settings -> Footer ->, this is also displayed in the Admin interface in the footer, but on the page where the users/whistleblowers are on the move is still the same footer, how can I update this?
Thx for assistance 
Nico

**Comment by evilaliv3** [comment]
Thank you for your question.

Is your setup a single tenant or a multitenant setup ?

**Comment by nicogithub22** [comment]
It is a single tenant setup

**Comment by evilaliv3** [comment]
In this case if you have configured and saved the footer configuration,
that should be visible on any interface.

> Message ID: ***@***.***>
>

Try to re open the application on a new tab to ensure application caches
are resetted

**Comment by nicogithub22** [comment]
It worked the first time I tried it about a month ago, but now I want to change something and the change is not applied.
Is there any way I can change or push this manually?

**Comment by evilaliv3** [comment]
We are not able to reproduce the issue at the moment.

> Message ID: ***@***.***>
>

If you can please upload some screenshots to show where it works and where
it does not.

Thank you

**Comment by nicogithub22** [comment]
You can see I added "Datenschutzerklärung" but is not visible in the "whistleblower-view"

![Inked2023-03-17 08_27_28-Site settings – Mozilla Firefox](https://user-images.githubusercontent.com/125379623/225841546-a581d300-1c6c-4e0e-8850-ec04b3bc141a.jpg)
![Inked2023-03-17 08_27_44-Site whistleblower– Mozilla Firefox](https://user-images.githubusercontent.com/125379623/225841547-8dec0651-fb38-40fd-8662-0024d0af7bd2.jpg)

**Comment by evilaliv3** [comment]
Maybe its a caching problem, we are not aware of bug on this but i would try restarting the application.

P.s.: are you aware to be violating the [license](https://github.com/globaleaks/GlobaLeaks/blob/main/LICENSE) of GlobaLeaks by removing the "Powered by GlobaLeaks" attribution clause?
In order to continue to use the software it is fundamental that this terms are restored. Thank you.

**Comment by nicogithub22** [comment]
I have already restarted the application, but it still remains like this.
Is there any command I can do on the server that deletes the cache or something else?


P.s.: thanks for the information, I changed it.

**Comment by nicogithub22** [comment]
I was logged in with the admin account, but it didn't work.
Now I logged in with my personal account and my addition was not in the footer. So I added it, saved it and that's it. Everything is displayed correctly.

**Comment by evilaliv3** [comment]
Perfect! thank you for this update and thank you for correcting the licensing aspect.

closing the ticket then.
